src/test/java/guitests/SaveCommandTest.java

```diff
@@ -0,0 +1,35 @@
+//@@author A0148052L
+package guitests;
+
+import java.io.File;
+
+import org.junit.Rule;
+import org.junit.Test;
+import org.junit.rules.TemporaryFolder;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.SaveCommand;
+
+public class SaveCommandTest extends TaskListGuiTest {
+
+    @Rule
+    public TemporaryFolder testFolder = new TemporaryFolder();
+
+    @Test
+    public void save_file_successful() {
```

Comment:
There are various scenarios where saving can be successful. However, the featureUnderTest here should probably be saveFile because you can only save files and not anything else. As for test scenario, things such as absoluteFilePath, relativeFilePath, invalidFilePath can be examples.